examples/middleware/middleware.ts

```diff
@@ -1,7 +1,9 @@
 import { NextRequest, NextResponse, NextFetchEvent } from "next/server";
 import { clerkMiddleware } from "@clerk/nextjs/server";
 
-export function middleware(request: NextRequest, event: NextFetchEvent) {
+import { getCloudflareContext } from "@opennextjs/cloudflare";
+
+export async function middleware(request: NextRequest, event: NextFetchEvent) {
```

Comment:
I don't think this needs to be `async` ?

```suggestion
export function middleware(request: NextRequest, event: NextFetchEvent) {
```

Reply:
good spot 👍 , I must have added it before making `getCloudflareContext` sync and forgot to revert it later